From 67e41ae53953bd2e39a2c5dcc9dcb592c6c97203 Mon Sep 17 00:00:00 2001
From: Nicolas Toromanoff <nicolas.toromanoff@foss.st.com>
Date: Fri, 14 Mar 2025 15:15:50 +0100
Subject: [PATCH 1633/1695] drivers: stm32_ltdc: sync comment with code

Comment said "immediate", but code was "next vsync"

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 739d313a7087820601d9b0da141ea94d08762311 drivers: stm32_ltdc: sync comment with code]
Change-Id: Ifec523bd484a310adeb96d0dd3c769dcd754ca50
Signed-off-by: Nicolas Toromanoff <nicolas.toromanoff@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/444321
Tested-by: Nicolas TOROMANOFF <nicolas.toromanoff@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Nicolas TOROMANOFF <nicolas.toromanoff@st.com>
Domain-Review: Yannick FERTRE <yannick.fertre@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/drivers/stm32_ltdc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/drivers/stm32_ltdc.c b/core/drivers/stm32_ltdc.c
index 94f4ab975..e87e5c1e5 100644
--- a/core/drivers/stm32_ltdc.c
+++ b/core/drivers/stm32_ltdc.c
@@ -199,7 +199,7 @@ static TEE_Result stm32_ltdc_final(void *device)
 	/* Disable secure layer */
 	io_clrbits32(ldev->regs + LTDC_LXCR, LXCR_LEN);
 
-	/* Reload configuration immediately. */
+	/* Reload configuration at next vertical blanking. */
 	io_write32(ldev->regs + LTDC_LXRCR, LXCR_RCR_VBR);
 
 	ldev->end_of_frame = false;
-- 
2.47.2

